src/components/betting-chips/BettingControls.module.scss

```diff
@@ -1,25 +1,191 @@
 .bettingField {
-    width: 100%;
+  width: 100%;
+  display: flex;
+  background-color: var(--bg-red);
+  grid-row: 1 / 1;
+  grid-column: 1 / span 3;
+
+  .player,
+  .tie,
+  .banker {
+    text-align: center;
     display: flex;
-    margin-bottom: 1rem;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100%;
+    border: 4px solid rgb(70, 16, 18);
+    border-bottom: none;
+    color: var(--betting-font-color);
+
+    > p {
+      font-size: 2rem;
+      color: rgba(250, 250, 250, 0.6);
+      margin: 0;
+      margin-bottom: 0.1rem;
+    }
+  }
 
-    .player, .tie, .banker {
-        height: 10rem;
-        background: rgb(164, 54, 54);
-        border: 2px solid orange;
+  .fieldInfo {
+    margin-top: 1rem;
+    .name,
+    .stake {
+      margin: 0;
     }
 
-    .player {
-        flex-basis: 40%;
-        border-width: 2px 0 2px 2px;
+    .name {
+      font-size: 1.5rem;
     }
 
-    .tie {
-        flex-basis: 20%;
+    .stake {
+      color: rgba(250, 250, 250, 0.5);
     }
+  }
+
+  .player {
+    flex-basis: 40%;
+    border-width: 4px 0 0 0;
+  }
+
+  .tie {
+    flex-basis: 20%;
+  }
+
+  .banker {
+    flex-basis: 40%;
+    border-width: 4px 0 0 0;
+  }
+}
+
+.controls {
+  padding: 0 1rem 1rem 1rem;
+  display: flex;
+  align-items: baseline;
+  justify-content: center;
+  gap: 0.5rem;
+  grid-row: 3 / 3;
+  grid-column: 1 / span 3;
+  z-index: 300;
+}
+
+.small {
+  height: 3rem;
+  width: 3rem;
+  font-size: 1.1rem;
+}
+
+.big {
+  width: 5rem;
+  height: 5rem;
+  font-size: 1.5rem;
+}
+
+.btn {
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: radial-gradient(
+      ellipse farthest-corner at right bottom,
+      #fedb37 0%,
+      #fdb931 8%,
+      #9f7928 30%,
+      #8a6e2f 40%,
+      transparent 80%
+    ),
+    radial-gradient(
+      ellipse farthest-corner at left top,
+      #ffffff 0%,
+      #ffffac 8%,
+      #d1b464 25%,
+      #5d4a1f 62.5%,
+      #5d4a1f 100%
+    );
+  border-radius: 100%;
+  border: none;
```

Comment:
I see these styles for every `button` HTML element you have. You can move common styles to `index.scss`

```css
button {
  border: none;
  outline: none;
}
```